builtin/files/cluster.yaml.tmpl

```diff
@@ -1179,13 +1179,25 @@ kubernetes:
     enabled: false
 
 #  controllerManager:
+#   resources:
+#     requests:
+#       cpu: 100m
+#       memory: 100Mi
+#     limits:
+#       cpu: 250m
+#       memory: 512Mi
+
+#  apiServer:
+#    # Memory limit for apiserver in MB (used to configure sizes of etcd caches, etc.)
+#    targetRamMb: 320Mi
```

Comment:
What did you set your target ram mb to?  320Mi sounds quite small?

Reply:
Bad copy and paste, we use 4096Mi, 320Mi comes from test values used by `kops`. I'll update this.